src/main/java/com/icthh/xm/ms/entity/service/ChainedLifecycleLepStrategy.java

```diff
@@ -32,6 +33,7 @@
     @Lazy
     private ChainedLifecycleLepStrategy internal;
 
+    @IgnoreLogginAspect
```

Comment:
Maybe in this case we can just replace service annotation to component annotation.
This class to really service.